agenta-cli/agenta/sdk/agenta_init.py

```diff
@@ -53,73 +41,64 @@ def init(
         app_id: Optional[str] = None,
         host: Optional[str] = None,
         api_key: Optional[str] = None,
+        config_fname: Optional[str] = None,
```

Comment:
Could you please update one of the examples to demonstrate how `config_fname` is set? The documentation mentions it's the path to the configuration file, but it doesn't clarify if it's a relative or absolute path.

Reply:
done